cityofaustin/vision-zero
# Ability to swap primary and secondary address

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/19016

## Testing
**URL to test:** 
[Netlify](https://deploy-preview-1551--atd-vze-staging.netlify.app/#/)


**Steps to test:**
1. Go to the crash details page, check out the new swap address button.
2. Click on it, confirm, watch the addresses swap. Also test cancelling.
3. There is a bug where if there is a null in one of the fields and not null value in the other field, on swap both of the primary and secondary will retain the not null value instead of one of them being null. This is because of how the layer data model is set up to ignore nulls in the edits table. We are addressing this in later work
4. Test a read only user, the button should not render


---
#### Ship list
